sdk/storage/storage-blob/test/blobbatch.spec.ts

```diff
@@ -511,17 +512,20 @@ describe("BlobBatch", () => {
     }
   });
 
-  it("submitBatch should work with multiple types of credentials for subrequests", async function() {
+  it("submitBatch should work with multiple types of credentials for subrequests", async function(this: Context) {
     recorder.skip(
       undefined,
       "UUID is randomly generated within the SDK and used in the HTTP request and cannot be preserved."
     );
     // Try to get serviceURL object with TokenCredential
     // when ACCOUNT_TOKEN environment variable is set
     let tokenCredential;
+    /* eslint no-empty: ["error", { "allowEmptyCatch": true }] */
```

Comment:
Why don't we want the test to fail if we cannot get the token credential?

Reply:
Updated the test case.